src/pages/aclmanagement.html

```diff
@@ -164,7 +166,11 @@ <h1 id="acl-management-view-title">
                                         <div class="small" ng-if="ruleData.role.$touched && !ruleData.role.$valid">
                                             <small>{{'acl_management.errors.role_length_too_short' | translate}}</small>
                                         </div>
+                                        <div class="small float-lg-left prefix-warning-text" ng-if="rule.role.startsWith(doublePrefix) && !showPrefixWarningIcon">
+                                            <small>{{'acl_management.rulestable.prefix_warning.text' | translate}}</small>
+                                        </div>
                                         <em></em>
+                                        <em ng-if="rule.role.startsWith(doublePrefix) && showPrefixWarningIcon" class="icon-warning" gdb-tooltip="{{'acl_management.rulestable.prefix_warning.text' | translate}}" tooltip-placement="right"></em>
```

Comment:
`ng-if="rule.role.startsWith(doublePrefix) && showPrefixWarningIcon"` I don't get this. Can the role name have a double `custom` prefix without having a single one before that?